PGQuestionGenerator.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
+const faker = require('faker')
+const argv = require('yargs').argv
+
+const lines = argv.lines || 10
+const filename = argv.output || 'PGQuestions.csv'
+const stream = fs.createWriteStream(filename)
+const imageFilePath = 'https://ghrsea13-sdc-group2.s3-us-west-2.amazonaws.com/';
+var j = 999;
```

Comment:
just curious why you counted down in this one and up in the other one for j - doesn't make a different, I was just wondering!